sys/include/xtimer/tick_conversion.h

```diff
@@ -12,22 +13,85 @@
  * @{
  * @file
  * @brief   xtimer tick <-> seconds conversions for different values of XTIMER_HZ
+ *
  * @author  Joakim Nohlgård <joakim.nohlgard@eistec.se>
+ * @author  Josua Arndt <jarndt@ias.rwth-aachen.de>
  */
-
 #ifndef XTIMER_TICK_CONVERSION_H
 #define XTIMER_TICK_CONVERSION_H
 
 #ifndef XTIMER_H
 #error "Do not include this file directly! Use xtimer.h instead"
 #endif
 
+#include <stdio.h>
+#include <string.h>
+#include "fmt.h"
 #include "div.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif
 
+/*
+ * As tick_conversion.h is included in multiple files ENABLE_DEBUG can not be
+ * used as it gets redefined "[-Werror]"
+ * Activate the debug print and enable module fmt in the project makefile as follow
+ *
+ * USEMODULE += fmt
+ * CFLAGS += -DENABLE_DEBUG_PRINT_TICK_CONVERSION=1
+ * */
+#ifndef ENABLE_DEBUG_PRINT_TICK_CONVERSION
+#define ENABLE_DEBUG_PRINT_TICK_CONVERSION   (0)
+#endif
+
+#if ENABLE_DEBUG_PRINT_TICK_CONVERSION
+#ifndef MODULE_FMT
+#error Module fmt is necessary to ENABLE_DEBUG_TICKS
+/* Add module fmt to the makefile.
+ * USEMODULE += fmt
+ */
+#endif
+#endif
+
+inline void debug_print_ticks32(uint32_t clock_hz, char *sign, uint32_t from,
+                         char *base_from, uint32_t to,char *base_to) {
```

Comment:
indention looks of, and please apply RIOT coding style see https://github.com/RIOT-OS/RIOT/wiki/Coding-conventions